contracts/src/instances/ezeth/ezETHHyperdrive.sol

```diff
@@ -0,0 +1,85 @@
+///     ############   XXXXX        ++0+        +0++        XXXXX   ###########
+///   ##////////////########       ++00++      ++00++       ########///////////##
+///  ##////////////##########      ++000++    ++000++      ##########///////////##
+///  ##%%%%%%/////      ######     ++0000+    +0000++     ######     /////%%%%%%##
+///    %%%%%%%%&&             ##   ++0000+    +0000++   ##           &&%%%%%%%%%
+///         %&&&                ##  +o000+    +000o+  ##              &&&%
+///                              ## ++00+-    -+00++ ##
+///                               #% ++0+      +0++ %#
+///                               ###-:Oo.++++.oO:-###
+///                                ##: 00++++++00 :##
+///                    #S###########* 0++00+++00++0 *##########S#
+///                  #S               % $ 0+++0 $ %              S#
+///                #S   ----------   %+++++:#:+++++%-----------    S#
+///              #S   ------------- %++++: ### :++++%------------    S#
+///             S    ---------------%++++*\ | /*++++%-------------     S
+///           #S     --------------- %++++ ~W~ ++++%666--o UUUU o-      S#
+///         #S?      ---------------  %+++++~+++++%&&&8 o  \  /  o       ?S#
+///        ?*????**+++;::,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,::;+++**????*?
+///      #?+////////////////////////////////////////////////////////////////+?#
+///    #;;;;;//////////////////////////////////////////////////////////////;;;;;#
+///  S;;;;;;;;;//////////////////////////////////////////////////////////;;;;;;;;;S
+/// /;;;;;;;;;;;///////////////////////////////////////////////////////;;;;;;;;;;;;\
+/// |||OOOOOOOO||OOOOOOOO=========== __  ___        ===========OOOOOOOO||OOOOOOOO|||
+/// |||OOOOOOOO||OOOOOOOO===========|  \[__ |   \  /===========OOOOOOOO||OOOOOOOO|||
+/// |||OOOOOOOO||OOOOOOOO===========|__/[___|___ \/ ===========OOOOOOOO||OOOOOOOO|||
+/// ||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
+/// |||////////000000000000\\\\\\\\|:::::::::::::::|////////00000000000\\\\\\\\\\|||
+/// SSS\\\\\\\\000000000000////////|:::::0x666:::::|\\\\\\\\00000000000//////////SSS
+/// SSS|||||||||||||||||||||||||||||:::::::::::::::||||||||||||||||||||||||||||||SSS
+/// SSSSSSSS|_______________|______________||_______________|______________|SSSSSSSS
+/// SSSSSSSS                                                                SSSSSSSS
+/// SSSSSSSS                                                                SSSSSSSS
+///
+/// @author DELV
+/// @title EzETHHyperdrive
+/// @notice A Hyperdrive instance that uses ezETH as the yield source.
+/// @custom:disclaimer The language used in this code is for coding convenience
+///                    only, and is not intended to, and does not, have any
+///                    particular legal or regulatory significance.
+contract EzETHHyperdrive is Hyperdrive, EzETHBase {
+    address constant ETH = 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
```

Comment:
We already have a constant for this in `contracts/src/libraries/Constants.sol`. Let's use that instead (see the `StETHHyperdrive` implementation.